target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
+						label = "logic_config";
+						reg = <0x0 0x20000>;
+						read-only;
+					};
+
+					partition@0x20000 {
+						label = "logic_addr";
+						reg = <0x20000 0x20000>;
+						read-only;
+					};
+
+					partition@0x40000 {
+						label = "bootmonitor";
+						reg = <0x40000 0x20000>;
+						read-only;
+					};
+
+					partition@0x80000 {
+						label = "flash_config";
+						reg = <0x80000 0x40000>;
+						read-only;
+					};
+
+					partition@0xc0000 {
+                                                label = "uboot";
+                                                reg = <0xc0000 0x40000>;
+                                        };
+
+					partition@0x100000 {
+                                                label = "uboot-env";
+                                                reg = <0x100000 0x20000>;
+                                        };
+					
+					partition@0x120000 {
+                                                label = "firmware";
+                                                reg = <0x120000 0x1EC0000>;
+                                        };
+
+					partition@0x1FE0000 {
+                                                label = "GPHY_CPR";
+                                                reg = <0x1FE0000 0x20000>;
+                                        };
+
+				};
+			};
+		};
+
+		gpio: pinmux@E100B10 {
+			pinctrl-names = "default";
+			pinctrl-0 = <&state_default>;
+
+			state_default: pinmux {
+				mdio {
+					lantiq,groups = "mdio";
+					lantiq,function = "mdio";
+				};
+				i2c {
+                                 	lantiq,pins = "io19", "io20"; /* port i2c */
+                                 	lantiq,open-drain;
+                                 	lantiq,pull = <2>;
+                         	};
+				 gphy-leds {
+                                        lantiq,groups = "gphy0 led0", "gphy0 led1",
+                                                        "gphy0 led2", "gphy1 led0",
+                                                        "gphy1 led1", "gphy1 led2";
+                                        lantiq,function = "gphy";
+                                        lantiq,pull = <2>;
+                                        lantiq,open-drain = <0>;
+                                        lantiq,output = <1>;
+                                };
+			};
+		};
+
+		ifxhcd@E101000 {
+			status = "okay";
+			/* gpios = <&gpio 14 0>; */
```

Comment:
Why are the gpios added as comment? Are they required to enable/disable usb-power to drive external harddrives for example?